web/README.md

```diff
@@ -25,60 +25,78 @@ git clone https://github.com/electric-sql/examples
 cd examples/web
 ```
 
-Install the dependencies, either using Yarn:
+Install the dependencies:
 
 ```sh
 yarn
 ```
 
-Or using npm:
-
-```sh
-npm install
-```
-
 ## Configure
 
-As a local-first system, ElectricSQL defaults to working without a backend. So you can run this example application without connecting to a replication service. However, to run with replication working, you'll need to configure a backend to connect to.
-
-The simplest way is to [sign up to ElectricSQL](https://console.electric-sql.com/auth/signup) and create an application. This will give you a globally unique `app` slug. Paste this into your [application config](https://electric-sql.com/docs/usage/configure) in [`./electric-config.js`](./electric-config.js).
+[Sign up to ElectricSQL](https://console.electric-sql.com/auth/signup) and create an application. This will give you a globally unique `app` identifier. Paste this into the `config` object in `electric-config.js`.
 
 Alternatively, see [these instructions](https://github.com/electric-sql/examples#running-the-backend-locally) to run and connect to the backend locally.
 
-## Run
+## Synchronise schema
+
+[Install the ElectricSQL CLI](), e.g.:
+
+```sh
+brew install electric-sql/tap/electric
+```
 
-Build:
+Login on the command line with your console credentials:
 
 ```sh
-yarn build
+electric auth login YOUR_EMAIL
+```
+
+Build and sync migrations (replacing `APP_ID` with your `app` identifier):
```

Comment:
Yup. One note is the migrations folder can be there, but the problem with the *pre-bundled* `./migrations/dist/index.js` atm is that it includes the `app` ID in the metadata. Perhaps we can resolve an issue here by e.g.: supporting an `'*'` value for the app ID in the bundle metadata, which the server then accepts for any app?

> If we remove the migrations dir from the examples, currently there is no way of playing out with the demos locally.

Can I clarify why -- is it that the CLI requires auth+init to be able to bundle and that means you need a cloud account?

Certainly we 💯 need an open source, local dev solution that doesn't require the cloud service. Be nice to fix the goalposts on it too -- so if the actual solution is extracting the core services (auth, migrations) from the console into dependencies that can be run locally as well as pulled into a console umbrella, perhaps we just do that and do it properly, so it supports (a) local dev (b) testability and (c) we can fix the config semantics.

Reply:
> Can I clarify why 
It's because applying migrations on local env requires the postgres files. The examples only work because those files are still there.

I think the way forward is to be able to run the console as part of the local dev env and then run migrations as expected. Don't know if it is a lot of work or not.